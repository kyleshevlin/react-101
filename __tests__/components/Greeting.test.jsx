import React from 'react'
import Greeting from '../../src/js/components/Greeting'

describe('Greeting', () => {
  it('should render correctly', () => {
    const component = shallow(<Greeting />)

    expect(component).toMatchSnapshot()
  })

  it('should render with the provided message', () => {
    const message = 'React 101'
    const component = render(<Greeting message={message} />)

    expect(component.text()).toEqual(message)
  })
})
