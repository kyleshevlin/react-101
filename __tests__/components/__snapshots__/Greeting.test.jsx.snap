// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting should render correctly 1`] = `
<h1>
  Hello world!
</h1>
`;
